### DISCUSSION TITLE
[Question] How to enabled the "automatic back button in AppBar" mention in docs?

### DISCUSSION THREAD

**Discussion by seriaati** [flet-dev/flet#3923] [Q&A]
### Question

In the docs: https://flet.dev/docs/controls/page/#on_view_pop it was mentioned that there is an automatic back button in the AppBar, how to enable it?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by tanmay-bhatgare** [comment]
You don’t need to enable it manually. If you’ve correctly implemented the `page.on_view_pop` function and your `appbar` doesn’t have a custom leading argument, Flet will automatically add the back button for you. Code might help to get your issue solved quickly.
